bears/documentation/DocumentationStyleBear.py

```diff
@@ -97,6 +99,13 @@ def process_documentation(self,
 
             new_metadata.append(m._replace(desc=new_desc.lstrip(' ')))
 
+        # Check for raw doctrings in python
+        if re.match(r'\s*r', doc_comment.indent):
```

Comment:
I'm not sure if this is so super-nice exploiting the indentation... This highly depends on parsing of indentation... Can we maybe introduce new markers for python also including `r"""`, `r"`, `r'` and `r'''`?

Reply:
I tried that but that did not solve the problem here( I really don't know why). The first method i tried was to change markers in https://github.com/coala/coala/blob/master/coalib/bearlib/languages/documentation/default.coalang and https://github.com/coala/coala/blob/master/coalib/bearlib/languages/documentation/doxygen.coalang  Plus then this issue would better be in `coala/coala` repo. What should I do? @Makman2